tests/test_cli.py

```diff
@@ -190,10 +193,66 @@ def test_invalid_lang(self, sample_pdf: str) -> None:
         result = self.run_cli([sample_pdf, "--ocr-lang", "piglatin"])
         result.assert_failure()
 
+    @pytest.mark.reference_generator
     @for_each_doc
-    def test_formats(self, doc: Path) -> None:
-        result = self.run_cli(str(doc))
+    def test_regenerate_reference(self, doc: Path) -> None:
+        reference = (doc.parent / "reference" / doc.stem).with_suffix(".pdf")
+
+        result = self.run_cli([str(doc), "--output-filename", str(reference)])
+        result.assert_success()
+
+    @for_each_doc
+    def test_formats(self, doc: Path, tmp_path_factory: pytest.TempPathFactory) -> None:
+        reference = (doc.parent / "reference" / doc.stem).with_suffix(".pdf")
+        destination = tmp_path_factory.mktemp(doc.stem).with_suffix(".pdf")
+
+        result = self.run_cli([str(doc), "--output-filename", str(destination)])
         result.assert_success()
+        # When needed, regenerate the reference PDFs by uncommenting the following line:
+        # reference.parent.mkdir(parents=True, exist_ok=True)
+        # shutil.copy(destination, reference)
```

Comment:
This can be removed, right?

Reply:
Right! Did so in c28b58fe69a99acb2283fd13bbc791d5471e0438